CODEOWNERS

```diff
@@ -48,12 +48,12 @@ src/cpp/ @kroening @tautschnig @peterschrammel
 
 # These files change frequently and changes are low-risk
 
-src/util/irep_ids.def @diffblue/cbmc-developers
+src/util/irep_ids.def
 
-unit/ @diffblue/cbmc-developers
-regression/ @diffblue/cbmc-developers
-jbmc/unit/ @diffblue/cbmc-developers
-jbmc/regression/ @diffblue/cbmc-developers
+unit/
+regression/
+jbmc/unit/
+jbmc/regression/
```

Comment:
Shouldn't those lines be removed completely, together with the comment above it?

Reply:
Removing the lines would have the effect of making those folders only match `*`, so the six people listed there would be code owners. The effect I'm trying to get is to make no one a code owner of those folders. How github will interpret this PR isn't very well documented, and the only way to find out is to try it. It's also possible I have the wrong syntax - they say it follows `.gitignore`, which is related to `gitattributes`.

Reply:
Could a group enlisting exactly those with write access be set up? Relying on undocumented or poorly documented behaviour is just bound to cause confusion.

Reply:
I think we actually want to keep the current behaviour, where no one is assigned as code owner for PRs that only edit those folders. The intention is to say "These files are so low risk that anyone with write access can approve them."I have added an extra comment to this PR making it more explicit what it is doing. I believe that this PR

Reply:
I think we actually want to keep the current behaviour, where no one is assigned as code owner for PRs that only edit those folders. The intention is to say "These files are so low risk that anyone with write access can approve them."

I have added an extra comment to this PR making it more explicit what it is doing.

This PR relies on poorly documented behaviour less than the previous state of affairs (assigning a group without write access to be code owners). I don't think that we shouldn't use an empty list of code owners just because the meagre documentation doesn't explicitly state that you can do that.